### PR TITLE
ODATA-1459: Undefined grouping properties

### PR DESCRIPTION
**Pull request by HeikoTheissen**
https://issues.oasis-open.org/browse/ODATA-1459

### CODE REVIEW

File: docs/groupKey.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function groupKey(groupingProperties, instance) {
```
Comment (ralfhandl) [inline docs/groupKey.js:1]: Do we have a unit test for this function?

Should we have executable code in our `docs` folder, or rather a [Jupyter notebook](https://github.com/n-riesco/ijavascript) with text and code and results intermingled, literate-programming style, following Donald E. Knuth's example?

Comment (HeikoTheissen) [reply docs/groupKey.js:1]: The code is not meant to be executed but serves documentary purposes only. And github.com offers nice features like history and edit-by-pull-request, would Jupyter also offer this?

Comment (HeikoTheissen) [reply docs/groupKey.js:1]: I've nevertheless added a test case, and it immediately spotted superfluous code.

**Comment by HeikoTheissen** [comment]
Agreed to close this with @ralfhandl and @GeraldKrause on 2022-01-14.

**Pull request closed without merging**